paasta_tools/paasta_maintenance.py

```diff
@@ -104,10 +114,71 @@ def is_safe_to_drain(hostname):
     return not are_local_tasks_in_danger()
 
 
-def are_local_tasks_in_danger():
+def is_healthy_in_synapse(local_port, backends):
```

Comment:
how about calling this ``is_healthy_in_haproxy`` ? I was kind of expecting this to be reading the synapse json file rather than the haproxy csv given its name.